Source/JavaScriptCore/bytecode/ExitKind.h

```diff
@@ -59,6 +59,7 @@ enum ExitKind : uint8_t {
     ExceptionCheck, // We exited because a direct exception check showed that we threw an exception from a C call.
     GenericUnwind, // We exited because we arrived at this OSR exit from genericUnwind.
     BigInt32Overflow, // We exited because of an BigInt32 overflow.
+    ResizableArrayBufferView, // We exited because we made an incorrect assumption about what type of ArrayBufferView we would see.
```

Comment:
How about naming this `BadArrayBufferViewType` instead?  Or `Unexpected ResizableArrayBufferView` if you want it to be really specific.

Reply:
We would like to make it specific since BadArrayBufferViewType can happen when we expect Int8Array and we got Uint16Array. We throw this when we expect this is not resizable and it turned out that this is resizable.

Reply:
Changed it to UnexpectedResizableArrayBufferView.